backend/README.md

```diff
@@ -0,0 +1,15 @@
+# TREY Järjestöportaali backend
+
+Azure Functions backend.
+
+# Prerequisites
+
+- [azure-functions-core-tools@4](https://learn.microsoft.com/en-us/azure/azure-functions/functions-run-local?tabs=macos%2Cisolated-process%2Cnode-v4%2Cpython-v2%2Chttp-trigger%2Ccontainer-apps&pivots=programming-language-typescript)
+- Docker for local development environment
+
+# Getting started
+
+```bash
+$ npm install
+$ npm start
+```
```

Comment:
Here could be some instructions to activate the correct node version via local `.nvmrc,` for example using `nvm`.